portal/models/app_text.py

```diff
@@ -212,24 +212,25 @@ def fetch_elements(url):
                 return ("[TESTING - fake response]", 'http://fake.org')
```

Comment:
You'll need to change what's returned here to fix the tests, let me know if you need any help :)